typescript/index.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
 
    "parserOptions": {
       "ecmaVersion": 2017,
-      "lib": ["ES2017"]
+      "lib": ["ES2017"],
```

Comment:
Bump this to ES2022 :)

Reply:
JS in on 2017 as well, should we bump both packages?

Reply:
I think so?

Reply:
Not sure if the [TS docs](https://www.typescriptlang.org/tsconfig#lib) are just out-of-date or not but they're listing 2021 as the latest supported version.